spec/features/profile/profile_create_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "Creating a profile" do
+  describe "after user registration" do
+    scenario "deletes his profile" do
```

Comment:
I think the scenario description is not valid. We are creating his profile, aren't we?